.github/workflows/release-ci.yaml

```diff
@@ -14,3 +14,123 @@ jobs:
       - run: |
           echo "Event: ${{ github.event_name }}" >> "$GITHUB_STEP_SUMMARY"
           echo "Ref: ${{ github.ref_name }}" >> "$GITHUB_STEP_SUMMARY"
+
+  build:
+    name: Build
+    uses: ./.github/workflows/build.yaml
+    secrets: inherit
+
+  check-is-release:
+    runs-on: ubuntu-latest
+    outputs:
+      is_release: ${{ steps.is_release_check.outputs.is_release }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.ref_name }}
+      - id: is_release_check
+        run: |
+          source scripts/ci/lib.sh
+          if is_release_version "${{ github.ref_name }}"; then
+            echo "is_release=true" >> "$GITHUB_OUTPUT"
+          else
+            echo "is_release=false" >> "$GITHUB_OUTPUT"
+          fi
+
+  check-scanner-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.ref_name }}
+      - name: Check scanner version
+        run: |
+          scripts/ci/lib.sh \
+            check_scanner_version
+
+  check-collector-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.ref_name }}
+      - name: Check collector version
+        run: |
+          scripts/ci/lib.sh \
+            check_collector_version
+
+  update-collector-released-versions:
+    needs: [check-scanner-version, check-collector-version, check-is-release]
+    if: needs.check-is-release.outputs.is_release == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Do we need it if we set `STACKROX_TAG`?

Reply:
We need the checkout to get the lib and dependent scripts. I am not sure if `fetch-depth: 0` is really required, but we use it for all our checkouts and I've seen workflows fail because only head was checked out and tags were looked at. Maybe something to consider for a clean up at some point. 

We pass `STACKROX_TAG` to the script to append it to a file in the collector repository and open a PR there, so it's irrelevant to the checkout.